src/components/message/message.twig

```diff
@@ -0,0 +1,21 @@
+{# 
+  @TODO
+  1. Generate an example of a message with a decorative icon.
+  #}
```

Comment:
This feels more relevant to the story file than this template to me.

Reply:
Would this be the appropriate comment style _and_ location for this comment within the (.mdx) story file?

![Screen Shot 2020-05-01 at 3 07 59 PM](https://user-images.githubusercontent.com/1427548/80845320-ae71e100-8bbd-11ea-8bc1-82441b49c47e.png)